superset/config.py

```diff
@@ -314,8 +314,7 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # AUTH_REMOTE_USER : Is for using REMOTE_USER from web server
 AUTH_TYPE = AUTH_DB
 
-# Uncomment to setup Full admin role name
-# AUTH_ROLE_ADMIN = 'Admin'
+AUTH_ROLE_ADMIN = "Admin"
```

Comment:
I don't think we need to uncomment this as it's the default in Flask-AppBuilder.

Reply:
Commented, and replaced ' with ".